routes/reps.js

```diff
@@ -0,0 +1,20 @@
+// For workoutlog data
+const express = require('express'); 
+const router = express.Router();
+const repsCtrl = require('../controllers/reps');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const repsCtrl = require("../controllers/reps");
```